packages/plugins/typescript/react-apollo/src/visitor.ts

```diff
@@ -80,8 +83,8 @@ export class ReactApolloVisitor extends ClientSideBaseVisitor<ReactApolloRawPlug
   }
 
   private _buildHocProps(operationName: string, operationType: string): string {
-    const typeVariableName = this.convertName(operationName + toPascalCase(operationType) + this._parsedConfig.operationResultSuffix);
-    const variablesVarName = this.convertName(operationName + toPascalCase(operationType) + 'Variables');
+    const typeVariableName = this._prefix + this.convertName(operationName + toPascalCase(operationType) + this._parsedConfig.operationResultSuffix);
```

Comment:
`convertName` accepts a second argument of options, which contains `prefix` field. I think it will be a better way to add prefix.

Reply:
I have tried passing some like 

```
    const typeVariableName = this.convertName(operationName + toPascalCase(operationType) + this._parsedConfig.operationResultSuffix, { prefix: this._prefix });
    const variablesVarName = this.convertName(operationName + toPascalCase(operationType) + 'Variables', { prefix: this._prefix });
```
but the result was

`ISchemaTypestestStringMutation` instead of `SchemaTypes.TestStringMutation`